src/Components/Auth/auth.js

```diff
@@ -7,9 +8,11 @@ const Auth = (props) => {
 	const [userPassword, setUserPassword] = useState("");
 	const [isLoggedIn, setIsLoggedIn] = useState(true);
 
+	const context = useContext(AuthContext);
+
 	const submitHandler = () => {
 		if (
-			userName.trim().length === 0 ||
+			// userName.trim().length === 0 ||
 			userEmail.trim().length === 0 ||
```

Comment:
why exactly are you trimming these strings to check length? does empty space count for length?